src/pages/home/report/OnboardingReportFooterMessage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, {useMemo} from 'react';
+import {withOnyx} from 'react-native-onyx';
+import type {OnyxCollection, OnyxEntry} from 'react-native-onyx';
+import Banner from '@components/Banner';
+import * as Expensicons from '@components/Icon/Expensicons';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as PolicyUtils from '@libs/PolicyUtils';
+import Navigation from '@navigation/Navigation';
+import * as ReportInstance from '@userActions/Report';
+import type {OnboardingPurposeType} from '@src/CONST';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import type {Policy as PolicyType, Report} from '@src/types/onyx';
+
+type OnboardingReportFooterMessageOnyxProps = {
+    /** Saved onboarding purpose selected by the user */
+    choice: OnyxEntry<OnboardingPurposeType>;
+
+    /** Collection of reports */
```

Comment:
Which reports? All users reports?

Reply:
Yeah, the app is looking for an admin chat to show in the banner.

Reply:
There is another way: to the active policyID by `nvp_expensify_activePolicyID` and find an admin chat. Let me try.

Reply:
Done - 907167c.

Reply:
https://github.com/Expensify/App/assets/57314631/8ff3c88c-6d11-4e50-98ab-a8ebacd9c283